tensorflow/tools/xla_extract/tf_graph_to_xla_lib.cc

```diff
@@ -113,25 +260,32 @@ xla::HloModuleProto ExtractHloFromGraphDef(const GraphDef& in_graph,
   std::unique_ptr<GraphExecutionState> execution_state;
   s = GraphExecutionState::MakeForBaseGraph(&gdef, ges_options,
                                             &execution_state);
-  if (!s.ok())
-    LOG(ERROR) << "execution state creation failed: " << s.error_message();
+  if (!s.ok()) {
+    LOG(FATAL) << "execution state creation failed: " << s.error_message();
```

Comment:
can we change this to ERROR?  otherwise the error doesn't propagate through pytest apparently.

Reply:
ok